### PR TITLE
Change to record gRPC remote read chunk timer metrics

### PR DESCRIPTION
**Pull request by LuQQiu**
Record the gRPC remote read chunk timer metrics to get a better idea of remote RPC performance.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@apc999 A PR to see the direct vs remote performance and see the remote GRPC overhead, PTAL

**Comment by LuQQiu** [comment]
It's a testing PR, not a PR plan to merge to master

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -1037,18 +1037,10 @@ public MetricKey build() {
           .build();
 
   // Client metrics
-  public static final MetricKey CLIENT_BLOCK_READ_CHUNK =
-      new Builder("Client.BlockReadDataChunk")
+  public static final MetricKey CLIENT_BLOCK_READ_CHUNK_REMOTE =
+      new Builder("Client.BlockReadDataChunkRemote")
```
Comment (apc999) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:1041]: `CLIENT_BLOCK_READ_CHUNK_REMOTE` is inconsistent with `Client.BlockReadDataChunkRemote`
I will leave this up to you which one to choose

**Review by apc999 — APPROVED** [review]
One comment left otherwise LGTM

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 1 above)
Comment (LuQQiu) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java:1041]: sorry will rebuilt from the master branch and redo the docGen

File: core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java [hunk 2]
```diff
@@ -486,16 +474,7 @@ private void readChunk() throws IOException {
...
-      if (mDetailedMetricsEnabled) {
-        try (Timer.Context ctx = MetricsSystem
-            .timer(MetricKey.CLIENT_BLOCK_READ_CHUNK.getName()).time()) {
-          mCurrentChunk = mDataReader.readChunk();
-        }
-      } else {
-        mCurrentChunk = mDataReader.readChunk();
-      }
-    }
+    mCurrentChunk = mDataReader.readChunk();
```
Comment (apc999) [inline core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java:477]: if (mCurrentChunk == null) ?

**Review by apc999 — CHANGES_REQUESTED** [review]
Check if we changed the logic?

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**